x11-misc/bbkeys/bbkeys-0.9.1-r1.ebuild

```diff
@@ -16,10 +18,10 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_install() {
-	emake DESTDIR="${D}" install || die
-	rm -rf "${D}"/usr/share/doc
-	dodoc AUTHORS BUGS ChangeLog NEWS README || die
+	emake DESTDIR="${ED%/}" install
```

Comment:
is this an Autoconf/Automake package (it looks like one)? Autoconf/Automake need to keep `emake DESTDIR="${D}" install`, as it includes the native handling for `--prefix`: https://wiki.gentoo.org/wiki/Project:Prefix/Technical_Documentation#The_variables_ED_and_EROOT